docs/source/cookbooks/evals.md

```diff
@@ -0,0 +1,124 @@
+# Evaluations
+
+The Llama Stack Evaluation flow allows you to run evaluations on your GenAI application datasets or pre-registered benchmarks.
+
+
+We introduce a new set of APIs in Llama Stack for supporting running evaluations of LLM applications.
```

Comment:
why "new"? Everything is either new or is already there :)

Reply:
removed "new" :)